pkgs/development/python-modules/serversyncstorage/default.nix

```diff
@@ -35,7 +37,9 @@ buildPythonPackage rec {
     pymysqlsa umemcache WSGIProxy requests pybrowserid
   ];
 
-  meta = {
-    broken = true; # 2018-11-04
+  meta = with stdenv.lib; {
+    description = "The SyncServer server software, as used by Firefox Sync";
+    homepage = "https://github.com/mozilla-services/server-syncstorage";
+    license = licenses.mpl20;
   };
```

Comment:
Would you like to maintain this, i.e. be added to `meta.maintainers`? This applies to all of the packages.

Reply:
Yes. I added myself to `meta.maintainers`.